internal/errors/errors_test.go

```diff
@@ -46,7 +45,7 @@ func TestErrTimeoutParseFailed(t *testing.T) {
 	}
 	tests := []test{
 		func() test {
-			wantErr := errors.New("invalid timeout value: 10hours\t:timeout parse error out put failed")
+			wantErr := New("invalid timeout value: 10hours\t:timeout parse error out put failed")
```

Comment:
In the case of unit test for `internal/errors/errors`. we avoided to use vald's original errors implement as `want` because we'd like to test these functions.
IMO, all of change about `internal/errors/errors_test.go` should be reverted.

Reply:
I am not sure why the set team decided to use the standard errors package to test this function, ErrTimeoutParseFailed uses Errorf from the internal/errors package to generate errors Vald's internal/errors/New is not used inside ErrTimeoutParseFailed, so this is not really a test code that is suitable for the actual situation.
I will revert it this time, but I would like you as product owner to bring this matter back to the SET team for thorough discussion.

Reply:
@vankichi

Reply:
thx, what I'd like to say is that when, for example, we test the `New()` of `internal/errors/errors.go`, we should not use `New()` for creating want object cuz of having no meaning.